app/views/articles/show.html.erb

```diff
@@ -1,13 +1,6 @@
 <h1><%= @article.title %></h1>
 <p><%= @article.body %></p>
-<p>
-  Author:
-  <% if @article.user.nil? %>
-    "Deleted User"
-  <% else %>
-    <%= @article.user.username %>
-  <% end %>
-</p>
+<%= display_author(@article) %>
```

Comment:
To make things more reusable pass in a user instance. 

display_author(@article.user)

Reply:
Done